zone.go

```diff
@@ -0,0 +1,244 @@
+package dns
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"sort"
+	"strings"
+)
+
+// AddDomain adds origin to s if s is not already a FQDN.
+func AddDomain(origin, s string) string {
```

Comment:
I can see the benefit of these helpers. But take for instance `IsEdns0` in `default.go` - which I added at the time, because I thought: will be useful for some. People just code this out themselves. The a) don't expect such a function to be present or b) don't find it in the docs. In retrospect I think most functions in `default.go` should not have been added. I'm sorry, but I feel the same about these.